lib/basedriver/helpers.js

```diff
@@ -12,38 +12,51 @@ const ZIP_EXTS = ['.zip', '.ipa'];
 const ZIP_MIME_TYPE = 'application/zip';
 
 async function configureApp (app, appExt, mountRoot="Volumes", windowsShareUserName="", windowsSharePassword="") {
-  if (_.isNull(app) || _.isUndefined(app)) {
+  if (!_.isString(app)) {
     // immediately shortcircuit if not given an app
     return;
   }
 
-  let newApp = null;
+  let newApp = app;
   let shouldUnzipApp = false;
 
   // check if we're copying from a windows network share
-  if (_.startsWith(app, "\\")) {
-    app = await copyFromWindowsNetworkShare(app, appExt, mountRoot, windowsShareUserName, windowsSharePassword);
+  if (newApp.startsWith("\\")) {
```

Comment:
Does the native `startsWith` work with all the versions of Node that we support?

http://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-startsWith

Reply:
I assume babel should handle it

Reply:
@mykola-mokhnach Maybe. I'll check to confirm.

Reply:
Yeah, the polyfill covers it.